docs/blog/rendered/campaign-finance.ipynb

```diff
@@ -0,0 +1,2182 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #12.    raw = conn.read_csv(csv_path)
</pre><p>How about using <code>ibis.read_csv</code> instead of an explicit connection?  </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ibis-project/ibis/pull/5727/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>

Reply:
<p>Great! I thought I needed it so that the contributions and committees tables would be in the same DB, but I see that they still are with ibis.read_csv()</p>